ecs-logs-collector.sh

```diff
@@ -362,6 +363,41 @@ get_docker_logs() {
   ok
 }
 
+get_docker_config() {
+  try "collect system's Docker configuration"
+  dstdir="${info_system}/docker_config"
+  mkdir -p "${dstdir}"
+
+  # Find and collect system's supporting configuraton files.
+  for conf in /etc/sysconfig/docker /etc/sysconfig/docker-storage-setup /etc/default/docker; do
+    if [ -f "${conf}" ]; then
+      install -D "${conf}" "${dstdir}${conf}"
+    fi
+  done
+
+  # Collect the docker.service unit if the system is running systemd.
+  if command -v systemctl > /dev/null; then
```

Comment:
Worth "collecting" info about how docker was started in other init systems?

Reply:
I agree, let's open a separate issue on this rather than delay this push